.circleci/build.sh

```diff
@@ -0,0 +1,92 @@
+#!/bin/bash
+
+set -ex
+
+source ./env
+
+export MAX_JOBS=8
+
+SCCACHE="$(which sccache)"
+if [ -z "${SCCACHE}" ]; then
+  echo "Unable to find sccache..."
+  exit 1
+fi
+
+if which sccache > /dev/null; then
+  # Save sccache logs to file
+  sccache --stop-server || true
+  rm ~/sccache_error.log || true
+  SCCACHE_ERROR_LOG=~/sccache_error.log RUST_LOG=sccache::server=error sccache --start-server
+
+  # Report sccache stats for easier debugging
+  sccache --zero-stats
+fi
+
+PYTORCH_DIR=/tmp/pytorch
+XLA_DIR="$PYTORCH_DIR/xla"
+git clone --recursive --quiet https://github.com/pytorch/pytorch.git "$PYTORCH_DIR"
+cp -r "$PWD" "$XLA_DIR"
+
+cd $PYTORCH_DIR
+
+# Install ninja to speedup the build
+pip install ninja
+
+# Install Pytorch
+patch -p1 < xla/pytorch.patch
+DEBUG=1 python setup.py build develop
```

Comment:
The C++ tests might fail with a linker error (visibility issue with `c10::Half::from_bits`) when `DEBUG` is set. If they fail, we'll have to drop `DEBUG=1`.